lib/camp.js

```diff
@@ -775,8 +795,9 @@ function pathLikeUnit(serverMethod, statusCode) {
             req.data[key] = matched[key];
           }
           res.statusCode = statusCode;
-          if (callbacks[cbindex] !== undefined) {
-            callbacks[cbindex](req, res);
+          var cb = callbacks[cbindex] && callbacks[cbindex].callback;
+          if (cb) {
```

Comment:
If possible, I would prefer to have booleans, instead of implicit conversion, in statement conditions.